rosidl_generator_py/test/test_interfaces.py

```diff
@@ -209,11 +219,11 @@ def test_check_constraints():
         setattr(c, 'char_value', b'abc')
 
     c.up_to_three_int32_values = []
-    assert [] == c.up_to_three_int32_values
+    assert [] == c.up_to_three_int32_values.tolist()
```

Comment:
@dirk-thomas won't the list assignment right above break the test? Or are we using data descriptors or something?

Reply:
Each message field has a property setter to ensure the right data type is being passed and the value range is valid. That setter also ensures that in this case an `array.array` is being stored (and not a `list`).